### DISCUSSION TITLE
Hi I wanted to make the default page as Getting Started instead of home page

### DISCUSSION THREAD

**Discussion by syedshadab19** [squidfunk/mkdocs-material#4230] [Q&A] [answered]
Hi I wanted to make the default page as Getting Started instead of home page so that whenever the URL is clicked it should navigate to the getting started instead of home. please find the below snapshot showcasing the getting started in the menu bar I want the getting started page to be displayed when the URL "https://squidfunk.github.io/" is entered instead of the home page.

![image](https://user-images.githubusercontent.com/25971751/183814845-3bdfd562-b88b-4ffd-be02-a0c55ca069b2.png)

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #4098.
